core/tauri-build/CHANGELOG.md

```diff
@@ -116,8 +116,8 @@
 
 ## \[1.0.0-rc.1]
 
-- Remove `cargo:rerun-if-changed` check for non-existent file that caused projects to *always* rebuild.
-  - [65287cd6](https://www.github.com/tauri-apps/tauri/commit/65287cd6148feeba91df86217b261770fed34608) remove non-existent cargo rerun check ([#3412](https://www.github.com/tauri-apps/tauri/pull/3412)) on 2022-02-11
+- Remove `cargo:rerun-if-changed` check for nonexistent file that caused projects to *always* rebuild.
+  - [65287cd6](https://www.github.com/tauri-apps/tauri/commit/65287cd6148feeba91df86217b261770fed34608) remove nonexistent cargo rerun check ([#3412](https://www.github.com/tauri-apps/tauri/pull/3412)) on 2022-02-11
```

Comment:
this is a commit message, revert it to match the commit message in the git history